models/public/aclnet/aclnet.md

```diff
@@ -49,18 +49,32 @@ Audio, name - `input`, shape - `1,1,1,L`, format is `N,C,1,L` where:
 
 ### Original Model
 
-Sound classifier (see labels file, `<omz_dir>/data/dataset_classes/aclnet.txt`), name - `203`, shape - `1,53`, output data format is `N,C` where:
+Sound classifier (see labels file, `<omz_dir>/data/dataset_classes/aclnet_53cl.txt`), name - `203`, shape - `1,53`, output data format is `N,C` where:
 
 - `N` - batch size
 - `C` - Predicted softmax scores for each class in [0, 1] range
 
 ### Converted Model
 
-Sound classifier (see labels file, `<omz_dir>/data/dataset_classes/aclnet.txt`), name - `203`, shape - `1,53`, output data format is `N,C` where:
+Sound classifier (see labels file, `<omz_dir>/data/dataset_classes/aclnet_53cl.txt`), name - `203`, shape - `1,53`, output data format is `N,C` where:
 
 - `N` - batch size
 - `C` - Predicted softmax scores for each class in [0, 1] range
 
+## Download a Model and Convert it into OpenVINO™ Inference Engine Format
+
+You can download models and if necessary convert them into Inference Engine format using the OpenVINO™ [Model Downloader and other automation tools](../../../tools/downloader/README.md) as shown in the examples below.
+
+An example of using the Model Downloader:
+```
+python3 downloader.py --name <model_name>
+```
+
+An example of using the Model Converter:
+```
+python3 converter.py --name <model_name>
+```
```

Comment:
Repeating this in every model page seems questionable. Maybe it could be added to `models/public/index.md` instead?

Reply:
we actually already have this info (how to load and convert model) in automation tools description. Might be better (from one prospective) to remove examples at all. @andrew-zaytsev what do you think?

Reply:
I'd keep the examples in the model doc files. Looks like they are not going to be changed frequently, but they highlight it again that the user should convert the model before usage. Also, the user can see how to quickly convert the model in place without need to jump to another document.

Reply:
add <omz_dir>/tools/downloader a a path